src/assessments/page/test-steps/general-navigation.tsx

```diff
@@ -16,34 +16,42 @@ const generalNavigationDescription: JSX.Element = (
 const generalNavigationHowToTest: JSX.Element = (
     <div>
         <ol>
-            <li>Examine the target page to determine whether:</li>
-            <ol>
-                <li>The page is part of a multi-page website or web app.</li>
+            <li>
+                Examine the target page to determine whether:
                 <ol>
                     <li>
-                        If there are no other pages within the site or app, select <Term>Pass</Term>
-                        .
+                        The page is part of a multi-page website or web app.
+                        <ol>
+                            <li>
+                                If there are no other pages within the site or app, select{' '}
+                                <Term>Pass</Term>.
+                            </li>
+                        </ol>
                     </li>
-                </ol>
-                <li>The page is the result of, or a step in, a process.</li>
-                <ol>
                     <li>
-                        If the page is part of a process, select <Term>Pass</Term>.
+                        The page is the result of, or a step in, a process.
+                        <ol>
+                            <li>
+                                If the page is part of a process, select <Term>Pass</Term>.
+                            </li>
+                        </ol>
                     </li>
                 </ol>
-            </ol>
+            </li>
             <li>
                 Verify that the page provides two or more ways to locate pages within the site or
                 app, such as:
             </li>
-            <ol>
-                <li>Site maps</li>
-                <li>Site search</li>
-                <li>Tables of contents</li>
-                <li>Navigation menus or dropdowns</li>
-                <li>Navigation trees</li>
-                <li>Links between pages</li>
-            </ol>
+            <li>
```

Comment:
The extra `</li><li>` pair at 44/45 is causing a blank list item to appear:
![image](https://user-images.githubusercontent.com/1752950/73030994-c16f5b00-3def-11ea-85ef-2c498dcaa57f.png)

Reply:
Good catch